lib/lib.es5.d.ts

```diff
@@ -258,6 +258,7 @@ interface ObjectConstructor {
       * Returns the names of the enumerable properties and methods of an object.
       * @param o Object that contains the properties and methods. This can be an object that you created or an existing Document Object Model (DOM) object.
       */
+    keys<K extends string>(record: Record<K, any>): K[];
```

Comment:
You should use the same parameter name (`o`) so the JSDoc matches the declaration